client/src/components/profile/components/portfolio.css

```diff
@@ -147,3 +165,45 @@
   margin-inline-start: 10px;
   position: absolute;
 }
+
+@supports not (selector(:has)) {
+  .portfolio-container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .portfolio-container h3 {
+    grid-column: 1/ -1;
+    grid-row: 2;
+    margin-block: 1em;
+  }
+
+  .portfolio-container a {
+    grid-column: 1/ -1;
+    grid-row: 4;
+  }
+
+  @media (min-width: 995px) {
+    .portfolio-container {
+      grid-template-rows: minmax(65px, 0.1fr) minmax(125px, 1fr) 75px;
+    }
+
+    .portfolio-container {
+      grid-template-columns: 300px 1.2fr;
+    }
+
+    .portfolio-container h3 {
+      grid-column: 2;
+      grid-row: 1;
+    }
+
+    .portfolio-container p {
+      grid-column: 2;
+      grid-row: 2;
+    }
+
+    .portfolio-container a {
+      grid-column: 2;
+      grid-row: 3;
+    }
+  }
+}
```

Comment:
`has` selector isn't supported in Firefox, so we are using `selector` function that isn't supported in opera to tell the browser to display the old style